src/PluginNodes/Boiler2PluginNodes.cs

```diff
@@ -25,7 +25,7 @@ public class Boiler2PluginNodes : IPluginNodes
     private BaseDataVariableState _maintenanceIntervalMinutesNode;
     private BaseDataVariableState _overheatThresholdDegreesNode;
     private BaseDataVariableState _currentTempDegreesNode;
-    private BaseDataVariableState _pressureNode;
+    private BaseDataVariableState _overheatIntervalNode;

```

Comment:
Let's call this _overheatIntervalSecondsNode to be consistent with the others. Also, pls move to be below the maintenance interval